### DISCUSSION TITLE
Q about Git, Darker, and Mercurial

### DISCUSSION THREAD

**Discussion by jsuit** [akaihola/darker#372] [Q&A]
Sorry if this is a dumb question (I just found this repo a day ago), but I'm interested in using this with an existing code base, but this code base uses Mercurial, not Git. How much work would it take to get this repo working with Mercurial ("nearly impossible" could also be an answer).

**Comment by akaihola** [comment]
@jsuit, it's been a really long time since I used Mercurial, so I'm really not able to estimate the effort without diving deep into the details.

If a contributor or two step up to start implementing this, I'll be more than happy to help both in text discussions on GitHub and in on-line calls to guide them around the code base and design.
